pages/1.13/metrics/index.md

```diff
@@ -10,37 +10,61 @@ enterprise: false
+    - Application identification (e.g., for Marathon apps): application_name
+    - System identification: agent_id
+		Example: An administrator can identify the group of metrics based on the following tags: 
+        - `agent_id` - Detects a situation which is system-specific (e.g., a faulty disk).
+        - `framework_id` - Detects which Mesos frameworks are using the most resources in the system.
+        - `container_id` - Shows the same information except with per-container granularity.
+
+3. **Forwarding Metrics:** After you have collected the metrics from the applications and hosts, it can be forwarded to various metrics stores. Configure Telegraf plugin to forward the metrics data from the cluster.
+
+4. **Monitoring and Visualizing Metrics:** It is recommended to install the DC/OS Monitoring service to monitor and visualize metrics in your DC/OS cluster. See the [service documentation](/services/beta-dcos-monitoring/0.4.3-beta/operations/install/) for instructions on how to install and use the service.
+
+# Telegraf
+Metrics in DC/OS 1.12 and newer versions use [Telegraf](/1.13/overview/architecture/components/#telegraf) to collect and process data. Telegraf provides a plugin-driven architecture. The custom DC/OS plugins provide metrics on the performance of DC/OS workloads and DC/OS itself. Telegraf provides metrics from DC/OS cluster hosts, containers running on those hosts, and from applications running on DC/OS using the `StatsD` process. It is natively integrated with DC/OS. By default, it exposes metrics in Prometheus format from `port 61091` on each node, and in JSON format through the DC/OS [Metrics API](/1.13/metrics/metrics-api/). Telegraf is included in the DC/OS distribution and runs on every host in the cluster. 
 
-## Monitoring and Visualization
-It is recommended to install the DC/OS Monitoring service to monitor and visualize metrics in your DC/OS cluster. See the [service documentation](/services/beta-dcos-monitoring/0.4.3-beta/operations/install/) for instructions on how to install and use the service.
+## Using Telegraf
+Telegraf collects application and custom metrics through the `dcos_statsd` plugin. A dedicated `StatsD` server is started for each new task. Any metrics received by the `StatsD` server are tagged with the task name and its service name. The address of the server is provided by environment variables (`STATSD_UDP_HOST` and `STATSD_UDP_PORT`). 
+
+<p class="message--note"><strong>NOTE: </strong>When a task is completed, metrics that are emitted but not gathered by Telegraf will be discarded.</p>
+
+The metrics collected by `dcos_statsd` are gathered every 30 seconds. The task must run for at least 30 seconds to ensure a task's metrics are gathered.
+
+For more information about the list of metrics that are automatically collected by DC/OS, read [Metrics Reference](/1.13/metrics/reference/) documentation.
 
-## Upgrading from 1.11
```

Comment:
Let's remove this entire section - it is relevant to users upgrading 1.11 -> 1.12.

It is not true for users upgrading 1.12 -> 1.13